packages/boot/test/upgrading/test-upgrade-contracts.js

```diff
@@ -7,32 +7,15 @@ import { test as anyTest } from '@agoric/swingset-vat/tools/prepare-test-env-ava
 import { buildVatController } from '@agoric/swingset-vat';
 
 /**
- * @type {import('ava').TestFn<
- *   Awaited<ReturnType<typeof makeTestContext>>
- * >}
+ * @type {import('ava').TestFn<{}>}
```

Comment:
I thought this kind of comment got collapsed to one-line. Is that `prettier` rule not enforced here, or did I misunderstand the rule?

Reply:
That's only enabled on a few packages:

https://github.com/Agoric/agoric-sdk/blob/c6447955ab0e54fd429463c373830f1d15df25c9/package.json#L45-L52

I didn't continue because the rule made some thing ugly in other packages. I've wanted to find time to make a prettier plugin that just formats the TS portion of the JSDoc and leaves the rest alone.